content/work_out_income.md

```diff
@@ -173,7 +175,21 @@ You might pay more for:
 - energy and water bills - you may spend more time at home
 - care costs
 
-##If you have debt
+## Paying more into your pension pot
+
+If you don’t think you’ll have enough income in your retirement you could consider adding more to your pot before you retire. You could talk to a financial adviser about how to contribute more to your pot. 
+
+You can also find out about [ways to boost your pension](https://www.moneyadviceservice.org.uk/en/articles/ways-to-boost-your-pension-in-the-run-up-to-retirement) from the Money Advice Service.
+
+There’s a limit to how much you can pay into your pot and still get tax relief. You can pay in up to your [annual allowance](https://www.gov.uk/tax-on-your-private-pension/annual-allowance).
+
+## The State Pension
+
+You can push back when you start receiving the State Pension to help increase your income in retirement. This is called [deferring the State Pension](https://www.gov.uk/deferring-state-pension/what-you-may-get).
+
+You may be able to [top up your State Pension](https://www.gov.uk/statepensiontopup) if you don’t get the full amount.
```

Comment:
@alexkouz - should this read **"if you won’t get the full amount"**? It's something you have to do before you draw your state pension?